src/cpu/flexcpu/simple_dataflow_thread.cc

```diff
@@ -1312,6 +1314,7 @@ SDCPUThread::populateUses(shared_ptr<InflightInst> inst_ptr)
 {
     const StaticInstPtr static_inst = inst_ptr->staticInst();
 
+#if TRACING_ON
```

Comment:
This shouldn't be necessary. If you're getting errors about uninitialized variable, use `M5_VAR_USED`